docs/SRS/SRS.tex

```diff
@@ -640,72 +640,72 @@ \subsection{Functional Requirements}
 \medskip
 \noindent \textbf{Requirement ID:} FR1\\
 \textbf{Use Case:} 1\\
-\textbf{Description:} The system must provide a user-friendly interface for users to enter their email and password when creating an account.\\
-\textbf{Rationale:} This requirement ensures that users can easily provide their registration information.\\
-\textbf{Fit Criterion:} Users should be able to successfully enter their email and password without encountering errors during the registration process.\\
+\textbf{Description:} The system must provide a clear and intuitive interface for account creation, where users can easily enter their email and password.\\
+\textbf{Rationale:} A registration interface is crucial for reducing the entry barrier for new users, enhancing user satisfaction, and decreasing the likelihood of registration abandonment.\\
+\textbf{Fit Criterion:} The interface should allow users to enter their email and password and complete the registration in no more than three steps, with each step clearly delineated.\\
 
 \medskip
 \noindent \textbf{Requirement ID:} FR2\\
 \textbf{Use Case:} 2\\
-\textbf{Description:} The system must validate user credentials (email and password) when a user attempts to sign in.\\
-\textbf{Rationale:} This requirement is necessary to fulfill the authentication process for registered users.\\
-\textbf{Fit Criterion:} Users with valid credentials should be granted access to the application, while users with invalid credentials should receive an error message.\\
+\textbf{Description:} The system must securely validate user credentials like email and password during the sign-in process.\\
```

Comment:
i like the old version better feels less ambiguous lol

Reply:
resolved